.dependabot/config.yml

```
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: npm
```

```suggestion
  - package_manager: javascript
```
Based off of the docs here: https://dependabot.com/docs/config-file/#package_manager-required